packages/edit-site/src/classic-stylebook.scss

```diff
@@ -0,0 +1,21 @@
+@include wordpress-admin-schemes();
+
+#wpadminbar,
+#adminmenumain {
+	display: none;
+}
+#wpcontent {
+	margin-left: 0;
+}
+body.js #wpbody {
+	padding-top: 0;
+}
+body {
+	@include wp-admin-reset("#gutenberg-stylebook");
+}
```

Comment:
```suggestion
body.js.appearance_page_gutenberg-stylebook-static {
	@include wp-admin-reset("#gutenberg-stylebook");
}
```

It looks like we need to make our CSS more specific, otherwise the left padding will be added.

![mobile-left-padding](https://github.com/user-attachments/assets/2a264178-bd3f-4952-9e87-421cab23a852)